src/ml.ts

```diff
@@ -0,0 +1,151 @@
+import * as tf from "@tensorflow/tfjs";
+import { GestureData, XYZData } from "./gestures-hooks";
+import { Filter, mlFilters } from "./mlFilters";
+import { SymbolicTensor } from "@tensorflow/tfjs";
+
+export enum Axes {
+  X = "x",
+  Y = "y",
+  Z = "z",
+}
+
+export const mlSettings = {
+  duration: 1800, // Duration of recording
+  numSamples: 80, // number of samples in one recording (when recording samples)
+  minSamples: 80, // minimum number of samples for reliable detection (when detecting gestures)
+  updatesPrSecond: 4, // Times algorithm predicts data pr second
+  numEpochs: 80, // Number of epochs for ML
+  learningRate: 0.5,
+  includedAxes: [Axes.X, Axes.Y, Axes.Z],
+  includedFilters: new Set<Filter>([
+    Filter.MAX,
+    Filter.MEAN,
+    Filter.MIN,
+    Filter.STD,
+    Filter.PEAKS,
+    Filter.ACC,
+    Filter.ZCR,
+    Filter.RMS,
+  ]),
+};
```

Comment:
I removed [automatic classification bool in ml settings](https://github.com/microbit-foundation/ml-trainer/blob/148eb97a6cb56b359ee12a818c93d0ce1124fcb5/src/script/stores/mlStore.ts#L118) since that is always true